README.md

```diff
@@ -21,13 +21,17 @@ This will be rendered as:
 To add a blog post, create a new markdown file (`YYYY_your_title.md`) in the `src/content/blog/` directory.
 The following template post can be used to start from scratch, or check out one of the existing blog posts ([markdown example](src/content/blog/2023_members.md), [live render](https://unitary.fund/posts/2023_members/)).
 
-```
+````
 ---
 title: post title
 author: author one and author two
 day: 7
 month: 2
 year: 2023
+tags: 
+  - python
+  - simulator
+  - quantum
```

Comment:
```suggestion
  - qiskit
```

Reply:
"quantum" is not a great example as most of what we fund is. "qiskit" is a tag example for which  we can collect more meaningful groups.